### PR TITLE
adopt vmray pack

### PR DESCRIPTION
**Pull request by jthom-vmray**
## Contributing to Cortex XSOAR Content
Make sure to register your contribution by filling the [contribution registration form](https://forms.gle/XDfxU4E61ZwEESSMA)

**The Pull Request will be reviewed only after the contribution registration form is filled.**

## Status
- [ ] In Progress
- [x] Ready
- [ ] In Hold - (Reason for hold)

## Description
Adopt VMRay pack.

## Minimum version of Cortex XSOAR
- [x] 6.0.0
- [ ] 6.1.0
- [ ] 6.2.0
- [ ] 6.5.0

## Does it break backward compatibility?
   - [ ] Yes
       - Further details:
   - [x] No

## Must have
- [x] Tests
- [x] Documentation

### CODE REVIEW

File: Packs/VMRay/pack_metadata.json [hunk 1]
```diff
@@ -1,11 +1,11 @@
 {
     "name": "VMRay",
     "description": "Analyze files and URLs using the VMRay Platform for accurate threat intelligence and high-quality IOCs.",
-    "support": "xsoar",
-    "currentVersion": "1.1.4",
+    "support": "partner",
+    "currentVersion": "1.1.5",
     "author": "Cortex XSOAR",
```
Comment (mchasepan) [inline Packs/VMRay/pack_metadata.json:6]:
```suggestion
    "author": "VMRay",
```

File: Packs/VMRay/pack_metadata.json [hunk 2]
```diff
@@ -1,11 +1,11 @@
 {
     "name": "VMRay",
```
Comment (mchasepan) [inline Packs/VMRay/pack_metadata.json:2]: Do you want to call the pack `VMRay Analyzer`?

File: Packs/VMRay/README.md [hunk 3]
```diff
@@ -1 +1 @@
-*Note: Support for this pack will be moving to the partner around Oct 15, 2021.*
+*Note: Support for this pack moved to the Partner on December, 21, 2021. Please contact the Partner directly via the support link on the right.*
```
Comment (mchasepan) [inline Packs/VMRay/README.md:1]: This readme should have detailed content in it on pack value and your company (similar to the paid packs), please sync w Zac on your side for details.

**Comment by kgal-pan** [comment]
@jthom-vmray - Please address build failure requesting to add an author image to `Packs/VMRay/Author_image.png`. See the [docs](https://xsoar.pan.dev/docs/packs/packs-format#author_imagepng) about the requirements of this image.

Also, please [sign the CLA agreement](https://cla-assistant.io/demisto/content?pullRequest=16556) as well and address @mchasepan's suggestions.

File: Packs/VMRay/README.md [hunk 4]
```diff
@@ -1 +1,15 @@
-*Note: Support for this pack will be moving to the partner around Oct 15, 2021.*
```
Comment (kgal-pan) [inline Packs/VMRay/README.md:1]: Add 
*Note: Support for this pack moved to the Partner on December, 21, 2021. Please contact the Partner directly via the support link on the right.*

File: Packs/VMRay/ReleaseNotes/1_1_5.md [hunk 5]
```diff
@@ -0,0 +1,2 @@
+##### VMRay
+*Note: Support for this pack moved to the Partner on December, 21, 2021. Please contact the Partner directly via the support link on the right.*
```
Comment (ostolero) [inline Packs/VMRay/ReleaseNotes/1_1_5.md:2]: Make this a bit clearer and shorter, full line can show in the readme as @kgal-pan suggested :)
```suggestion
- Support moved to partner on December 21, 2021
```

**Review by ostolero — CHANGES_REQUESTED** [review]
Hey Thank you for making @mchasepan changes! Only two small changes that need your attention and then we can merge. :)

**Review by ostolero — APPROVED** [review]
Great! let's merge

**Pull request merged**